packages/grid/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -35,6 +35,9 @@ export const GridRootStyles = styled('div', {
       [`&.${gridClasses['root--disableUserSelection']} .${gridClasses.cell}`]:
         styles['root--disableUserSelection'],
     },
+    { [`& .${gridClasses.actionsCell}`]: styles.actionsCell },
+    { [`& .${gridClasses.actionsCellItem}`]: styles.actionsCellItem },
+    { [`& .${gridClasses.booleanCell}`]: styles.booleanCell },
```

Comment:
Noticed that `booleanCell` was missing here. There's probably a lot more of those.

Reply:
I wonder if we can automate this instead of adding each one manually.
Some of the classes use `&.` because they're applied to the root element, others use `& .` (notice the space) because they're not applied to the root element.
But still, we can do something like this:
```tsx
overridesResolver: (props, styles) => {
  Object.keys(gridClasses).map(key => {
    const prefix = isRootClassName ? '&' : '& ';
    return { [`${prefix}.${gridClasses[key]}`]: styles[key] }
  });
}
```